src/CMakeLists.txt

```diff
@@ -123,24 +123,42 @@ set(GISH_H
 )
 
 if(WIN32)
-    list(append GISH_H video/glext.h)
+    list(APPEND GISH_H video/glext.h)
 endif(WIN32)
 
 if(APPLE)
     list(APPEND GISH_SRCS sdl/SDLMain.m)
     list(APPEND GISH_H sdl/SDLMain.h)
 endif(APPLE)
 
-add_executable(freegish ${GISH_SRCS} ${GISH_H})
+if(WIN32)
+    add_executable(freegish WIN32 ${GISH_SRCS} ${GISH_H})
+else()
+    add_executable(freegish ${GISH_SRCS} ${GISH_H})
+endif()
 
-target_link_libraries(freegish
-    ${SDL_LIBRARY}
-    ${OPENAL_LIBRARY}
-    ${OPENGL_LIBRARIES}
-    ${Vorbis_LIBRARIES}
-    ${PNG_LIBRARIES}
-    ${M_LIBRARIES}
-)
+if(MSVC)
+    target_link_libraries(freegish PRIVATE
+        ${OPENGL_LIBRARIES}
+        SDL2::SDL2
+        SDL2::SDL2main
+        Ogg::ogg
+        Vorbis::vorbis
+        Vorbis::vorbisenc
+        Vorbis::vorbisfile
+        png
+        OpenAL::OpenAL
+    )
```

Comment:
The method used for Linux is also possible to be used for Windows if `find_package` without `CONFIG` is used.

Reply:
I did a test on Ubuntu 22.04 with the packages from the repository. They do not provide the Vorbis, OpenAL and Ogg link targets. find_package doesn't work with Ogg at all. Not sure if linking to ogg is actually required in Windows, I'll look into that, the README could have been wrong.

Reply:
I tried both Linux (NixOS) and Windows (using vcpkg), both work fine without Ogg. Music and sound effects are still working :+1: